README.md

```diff
@@ -250,6 +250,16 @@ There are various ways to bind application level data structures to CQL statemen
 * [gocassa](https://github.com/hailocab/gocassa) is an external project that layers on top of gocql to provide convenient query building and data binding.
 * [gocqltable](https://github.com/kristoiv/gocqltable) provides an ORM-style convenience layer to make CRUD operations with gocql easier.
 
+Training
```

Comment:
Maybe call it something else? Perhaps "Resources" or "More knowledge and examples" or something?